plugins/node/opentelemetry-instrumentation-pino/test/pino.test.ts

```diff
@@ -112,8 +113,11 @@ describe('PinoInstrumentation', () => {
       const span = tracer.startSpan('abc');
       instrumentation.setConfig({
         enabled: true,
-        logHook: (_span, record) => {
+        logHook: (_span, record, level) => {
           record['resource.service.name'] = 'test-service';
+          if (semver.compare(pino.version, '7.9.0') >= 0) {
```

Comment:
I think semver.satisfies is a bit easier to read

Reply:
Done